website/ref/language.md

```diff
@@ -1981,11 +1981,14 @@ Syntax:
 ```elvish-transcript
 unless <condition> {
     <body>
+} else {
+    <else-body>
 }
 ```
 
-The `unless` special command evaluates the condition to a booleanly true value
-and skips its body.
+The `unless` special command evaluates the condition to a booleanly true value,
+skips its body and executes the else body, if the else body is present. It is
+the negation of the `if` special command except no `elif` clauses are allowed.
```

Comment:
Ugh! This is another reason, IMHO, why this should not be introduced. At least in its current form. I can't see any rational reason why `elif` clauses should not be allowed since `unless` is simply syntactic sugar for `if (not *expression*)`. It seems like the restriction is because the syntactic sugar is not really an obvious, or equivalent, variant of `if (not *expression*)`. Which argues against its addition, IMHO.